public/lib/capi-atom-service.js

```diff
@@ -70,6 +73,10 @@ function wfCapiAtomService($http, $q, config, wfCapiContentService, wfAtomServic
         const atom = _.get(response.data.response[atomType].data, atomType);
         atom.defaultHtml = _.get(response.data.response[atomType], 'defaultHtml');
         atom.contentChangeDetails = _.get(response.data.response[atomType], 'contentChangeDetails');
+        atom.commissioningInfo = _.get(response.data.response[atomType].data[atomType], 'commissioningDesks').map(desk => {
```

Comment:
As this is a bit magical, maybe a comment to explain how the `commissioningDesks` in the response usually look would be useful